presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -1108,30 +1068,31 @@ public void createView(ConnectorSession session, SchemaTableName viewName, Strin
                 .put(PRESTO_VIEW_FLAG, "true")
                 .build();
 
-        FieldSchema dummyColumn = new FieldSchema("dummy", STRING_TYPE_NAME, null);
-
-        StorageDescriptor sd = new StorageDescriptor();
-        sd.setCols(ImmutableList.of(dummyColumn));
-        sd.setSerdeInfo(new SerDeInfo());
-
-        Table table = new Table();
-        table.setDbName(viewName.getSchemaName());
-        table.setTableName(viewName.getTableName());
-        table.setOwner(session.getUser());
-        table.setTableType(TableType.VIRTUAL_VIEW.name());
-        table.setParameters(properties);
-        table.setViewOriginalText(encodeViewData(viewData));
-        table.setViewExpandedText("/* Presto View */");
-        table.setSd(sd);
-
+        FieldSchema dummyColumn = new FieldSchema("dummy", HIVE_STRING, Optional.empty());
         PrivilegeGrantInfo allPrivileges = new PrivilegeGrantInfo("all", 0, session.getUser(), PrincipalType.USER, true);
-        table.setPrivileges(new PrincipalPrivilegeSet(
+
+        Table.Builder tableBuilder = new Table.Builder()
```

Comment:
I prefer builders to have a static `builder()` method instead of a public constructor